openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java

```diff
@@ -0,0 +1,968 @@
+				extraJvmOptions += " -Djava.ext.dirs=" + jckBase + File.separator + "lib" + File.separator + "extensions" + File.pathSeparator + 
+										testJdk + File.separator + "jre" + File.separator + "lib" + File.separator + "ext";
+			}
+			if (tests.contains("api/signaturetest")) {
+				fileContent += "set jck.env.runtime.staticsigtest.staticSigTestClasspathRemote \"" + getSignatureTestJars(pathToLib) + "\"" + ";\n";
+			}
+			if (extraJvmOptions.contains("nofallback") && tests.startsWith("vm") ) {
+				fileContent += "set jck.env.testPlatform.typecheckerSpecific No" + ";\n";		
+			}
+			extraJvmOptions += suppressOutOfMemoryDumpOptions;
+			
+			// Add the JVM options supplied by the user plus those added in this method to the jtb file option.
+			fileContent += "set jck.env.runtime.testExecute.otherOpts \" " + extraJvmOptions + " \"" + ";\n";
+		}
+		
+		// Compiler settings
+		if (testSuite == TestSuite.COMPILER) {
+			
+			keyword = "keywords compiler";
+			
+			if (platform.equals("win32")) {
+					pathToRmic += ".exe";
+					testPlatform = "Windows";
+					concurrency = "1";
+			} else if (platform.equals("linux") || platform.equals("mac")) {
+				    testPlatform = "Linux";
+					concurrency = "1";	
+			} else if (platform.equals("aix")) {
+			        testPlatform = "Linux";		
+					concurrency = "1";
+			} else if (platform.equals("zos")) {
+					pathToLib = testJdk + File.separator + "lib";
+					testPlatform = "Linux";
+					concurrency = "4";
+			} else {
+				    throw new StfException("Unknown platform:: " + platform);
+			}
+			
+			fileContent += "concurrency " + concurrency + ";\n";
+			fileContent += "timeoutfactor 1" + ";\n";							// lang.CLSS,CONV,STMT,INFR requires more than 1h to complete. lang.Annot,EXPR,LMBD require more than 2h to complete tests
+			fileContent += keyword + ";\n";
+			fileContent += "set jck.env.testPlatform.os " + testPlatform + ";\n";
+			fileContent += "set jck.env.compiler.testCompile.testCompileAPImultiJVM.cmdAsString \"" + pathToJava + "\"" + ";\n";
+			
+			if (tests.contains("api/java_rmi")) {
+				fileContent += "set jck.env.compiler.testRmic.cmdAsString \"" + pathToRmic + "\"" + ";\n";
+			}
+			
+			if (tests.contains("api/signaturetest")) {
+				fileContent += "set jck.env.compiler.testCompile.compilerstaticsigtest.compilerStaticSigTestClasspathRemote \"" + getSignatureTestJars(pathToLib) + "\"" + ";\n";
+			}
+			
+			fileContent += "set jck.env.compiler.compRefExecute.cmdAsString \"" + pathToJava + "\"" + ";\n";
+			fileContent += "set jck.env.compiler.testCompile.otherOpts \"-source 1.8 \"" + ";\n";
+
+			extraJvmOptions += suppressOutOfMemoryDumpOptions;
+			
+			// Add the JVM options supplied by the user plus those added in this method to the jtb file option.
+			fileContent += "set jck.env.compiler.compRefExecute.otherOpts \" " + extraJvmOptions + " \"" + ";\n";	
+		}
+		// Devtools settings
+		if (testSuite == TestSuite.DEVTOOLS) {
+		
```

Comment:
Could we fix the indentation in the if-else block below?

Reply:
Done